modules/manage/pages/iceberg/query-iceberg-topics.adoc

```diff
@@ -16,6 +16,11 @@ When you access Iceberg topics from a data lakehouse or other Iceberg-compatible
 == Access Iceberg tables
 
 ifndef::env-cloud[]
+[IMPORTANT]
+====
+Redpanda v25.3 introduces breaking schema changes for Iceberg topics. Before upgrading, review xref:manage:iceberg/iceberg-schema-changes-and-migration-guide.adoc[] and follow the required migration steps to avoid data loss or query errors.
```

Comment:
"Redpanda v25.3 introduces breaking schema changes for Iceberg Topics. If you are already using Iceberg Topics and want to maintain the data in the corresponding Iceberg tables, before upgrading your cluster, review xref:manage:iceberg/iceberg-schema-changes-and-migration-guide.adoc[] and follow the required migration steps to avoid sending new topic records to a Dead Letter Queue table."

Reply:
Agree, data isn't lost per se -- it is put into the DLQ

Reply:
Updated